packages/@glimmer/compiler/lib/passes/1-normalization/keywords/utils/equality.ts

```diff
@@ -0,0 +1,67 @@
+import { ASTv2, generateSyntaxError } from '@glimmer/syntax';
+
+import { Err, Ok, Result } from '../../../../shared/result';
+import * as mir from '../../../2-encoding/mir';
+import { NormalizationState } from '../../context';
+import { VISIT_EXPRS } from '../../visitors/expressions';
+import { GenericKeywordNode, KeywordDelegate } from '../impl';
+
+function assertEqualKeyword(node: GenericKeywordNode): Result<ASTv2.PositionalArguments> {
+  let {
+    args: { named, positional },
+  } = node;
+
+  if (named && !named.isEmpty()) {
+    return Err(generateSyntaxError(`(eq) does not take any named arguments`, node.loc));
+  }
+
+  return Ok(positional);
+}
```

Comment:
I think we also need to assert if you use more than two positional arguments

https://github.com/emberjs/rfcs/blob/master/text/0560-add-equality-operators.md#eq

> {{eq}}
> Binary operation. Throws an error if not called with exactly two arguments. Equivalent of === This is identical to the eq helper in ember-truth-helpers

Reply:
I can do that!  I built this in b/c I saw a comment from a certain someone ;). Happy to do whatever you feel best for this initial PR though.

https://github.com/emberjs/rfcs/pull/560#discussion_r385882319